app/src/main/java/org/oppia/app/player/state/StateAdapter.kt

```diff
@@ -143,10 +151,14 @@ class StateAdapter(private val itemList: MutableList<Any>, private val interacti
   }
 
   inner class ContentViewHolder(val binding: ViewDataBinding) : RecyclerView.ViewHolder(binding.root) {
-    internal fun bind(rawString: String?) {
+    internal fun bind(rawString: String) {
       binding.setVariable(BR.htmlContent, rawString)
       binding.executePendingBindings()
-      binding.root.content_text_view.text = rawString
+      val htmlResult: Spannable = htmlParserFactory.create( entityType,explorationId).parseOppiaHtml(
```

Comment:
Incorrect spacing. Update this simple by pressing: `Option+command+L`

Reply:
corrected.